node/tests/hubController.js

```diff
@@ -0,0 +1,45 @@
+const sleep = require('es6-sleep').promise;
+var test = require('ava');
+var config = require('./hubController-testConfig');
+
+var hubControllerPath = require('path').join(__dirname, '..');
+var HubController = require("../hubController");
```

Comment:
i do NameClass for class and nameClass for instance of object.

---
In reply to: [88996532](https://github.com/openT2T/cloud/pull/9#discussion_r88996532) [](ancestors = 88996532)